images/build/Dockerfiles/temporaryFiles.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM githubrunners-buildpackdeps-stretch
+ARG IMAGES_DIR=/tmp/oryx/images
+ARG BUILD_DIR=/tmp/oryx/build
+RUN mkdir -p ${IMAGES_DIR} \
+    && mkdir -p ${BUILD_DIR}
+COPY images ${IMAGES_DIR}
+COPY build ${BUILD_DIR}
+RUN find ${IMAGES_DIR} -type f -iname "*.sh" -exec chmod +x {} \; \
+    && find ${BUILD_DIR} -type f -iname "*.sh" -exec chmod +x {} \;
```

Comment:
can we rename this to buildcontext.dockerfiles or something like that?

Reply:
In my latest commit I avoided creating the name with `buildcontext` since there is actually a concept of docker build context and i did not want to confuse the name with that, so i created a different name `supportFilesForBuildingBuildImages.Dockerfile`